M2/Macaulay2/packages/TropicalToric.m2

```diff
@@ -0,0 +1,98 @@
+newPackage(
+  "TropicalToric",
+	Version => "1.0",
+	Date => "May 2022",
+	Authors => {
+   		{
+        Name => "Alessio Borzì",
+        Email => "Alessio.Borzi@warwick.ac.uk",
+        HomePage=>"https://alessioborzi.github.io"
+      }
+  },
+	Headline => "A package on tropical methods for toric intersection theory",
+	Configuration => {},
+  PackageExports => {
+    "NormalToricVarieties",
+    "Tropical",
+    "Package$gfanInterface"
+  },
+  PackageImports => {
+    "FourierMotzkin"
+  },
+	DebuggingMode => true,
+	AuxiliaryFiles => true,
+	CacheExampleOutput => true,
+  OptionalComponentsPresent => true,
+  UseCachedExampleOutput => true,
```

Comment:
Why is this using cached examples?

Reply:
Good question.  I just copied this verbatim from JSAG.  I don't think it needs it.

Reply:
I realized it technically needs Tropical, which needs Polymake. `OptionalComponentsPresent` should probably be transitive, no?

Reply:
Ah, ok.  Yes, that would definitely make sense.